src/main/java/com/yelp/nrtsearch/server/luceneserver/field/DateTimeFieldDef.java

```diff
@@ -97,11 +101,16 @@ public Query getRangeQuery(RangeQuery rangeQuery) {
     return new IndexOrDocValuesQuery(pointQuery, dvQuery);
   }
 
-  private static long convertDateStringToMillis(
-      String dateString, DateTimeFormatter dateTimeFormatter) {
-    return LocalDateTime.parse(dateString, dateTimeFormatter)
-        .toInstant(ZoneOffset.UTC)
-        .toEpochMilli();
+  private static long convertDateStringToMillis(String dateString, String dateTimeFormat) {
+    switch (dateTimeFormat) {
+      case EPOCH_MILLIS:
```

Comment:
I see switch-case used here and elsewhere for only 2 and sometimes 1 case including the default case. Are we planning to add more conditions? If not we can use if-else

Reply:
Sure, changed it to if-else